templates/landingpage.html.j2

```diff
@@ -41,37 +41,65 @@
                 </div>
         </header>
         <main>
-            <div class="creators">
-                <div class="row">
-                    <div class="col-sm-12">
-                        {% for creator in creators %} {{ creator.Name.Family_Name }}, {{ creator.Name.Given_Name }} {{ "&" if not loop.last }} {% endfor %}
+            <div>
+                <div class="row row-override">
+                    <div class="col-lg-8">
+                        <div class="creators">
+                            {% for creator in creators %} {{ creator.Name.Family_Name }}, {{ creator.Name.Given_Name }} {{ "&" if not loop.last }} {% endfor %}
+                        </div>
+                        <section class="content">
+                            <h2>{{ title }}</h2>
+                            <div class="meta">
+                                <label>Publication Date:</label>
+                                <span class="date">{{ publication_date }}</span>
+                                <label>Accessibility:</label>
+                                <span>{{ data_access_restriction }}</span>
+                            </div>
+                            <p class="description">{{ description }}</p>
+                            <div class="keywords">
+                                <label>Keywords</label>
+                                {% for tag in tags %}
+                                <span class="keyword">{{ tag }}</span>
+                                {% endfor %}
+                            </div>
+                        </section>
+                    </div>
+                    <div class="col-lg-4 pb-3 pt-3 version-div">
```

Comment:
We can rename version-div to just version and reuse it for all the subclasses. So we don't need version-title, version-a etc.